eth/api_backend.go

```diff
@@ -449,16 +453,20 @@ func (b *EthAPIBackend) StartMining(threads int) error {
 	return b.eth.StartMining(threads)
 }
 
-func (b *EthAPIBackend) StateAtBlock(ctx context.Context, block *types.Block, reexec uint64) (*state.StateDB, mps.PrivateStateRepository, func(), error) {
-	return b.eth.stateAtBlock(block, reexec)
+func (b *EthAPIBackend) StateAtBlock(ctx context.Context, block *types.Block, reexec uint64, base *state.StateDB, checkLive bool) (*state.StateDB, mps.PrivateStateRepository, error) {
+	state, privateState, err := b.eth.stateAtBlock(block, reexec, base, checkLive)
+	return state, privateState, err
```

Comment:
why create extra var while you can return directly also same is happening in [geth](https://github.com/ethereum/go-ethereum/blob/master/eth/api_backend.go#L362)
```suggestion
	return b.eth.stateAtBlock(block, reexec, base, checkLive)
```